openupgrade_scripts/scripts/account_edi/14.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,5 @@
+---Models in module 'account_edi'---
+---Fields in module 'account_edi'---
```

Comment:
The analysis work file is a copy paste of the analysis file with comment you could add.

Reply:
> The analysis work file is a copy paste of the analysis file with comment you could add.

@legalsylvain ok thank you bro. I understand.